Examples/WindowProperties/WindowPropertiesApp.swift

```diff
@@ -0,0 +1,31 @@
+import SwiftCrossUI
+
+class WindowPropertiesAppState: AppState {
+    @Observed var count = 0
+}
+
+@main
+struct WindowPropertiesApp: App {
+    let identifier = "dev.stackotter.WindowPropertiesApp"
+    
+    let state = WindowPropertiesAppState()
+    
+    /// This example uses all available window properties to show how a window can be customized.
+    let windowProperties = WindowProperties(title: "WindowPropertiesApp",
+                                            defaultWidth: 500,
+                                            defaultHeight: 650,
+                                            resizable: false)
+    
+    var body: some ViewContent {
+        HStack {
+            VStack {
+                Text("") /// Placeholder until .padding() is available
```

Comment:
Comments within a function should only be two slashes, three slashes is a documentation comment for a declaration